### DISCUSSION TITLE
Running this project.

### DISCUSSION THREAD

**Discussion by Natasha-Zero** [StoicNeutron/GMU_ACM#11] [Q&A]
M a bit new here! I ran nom run dev but I don't work. Could anyone help me with this?

**Comment by StoicNeutron** [comment]
I suppose u mean npm not nom. Make sure you have all the dependencies install, example: node module. Install them first, then u'll be able to run it. Note that the route isn't the default http://localhost:3000/, it http://localhost:3000/home or http://localhost:3000/about. Hope this help.
